lld/test/ELF/keep-text-section-prefix.s

```diff
@@ -0,0 +1,89 @@
+# REQUIRES: x86
```

Comment:
Why is this x86 only? The assembly below is consumed with the x86 linux triple so it shouldn't matter?

Reply:
We use x86 as a placeholder for target-agnostic behavior. llvm-mc depends on llvm/lib/Target/X86, and we use `REQUIRES: x86` to prevent test execution when LLVM_TARGETS_TO_BUILD skips x86.